docker-images/openemr/Dockerfile

```diff
@@ -17,14 +17,14 @@ ENV APACHE_LOG_DIR=/var/log/apache2
 COPY php.ini /etc/php7/php.ini
 COPY openemr.conf /etc/apache2/conf.d/
 #add runner and auto_configure and prevent auto_configure from being run w/o being enabled
-COPY run_openemr.sh .
-COPY auto_configure.php .
-RUN chmod 700 run_openemr.sh \
-    && chmod 000 auto_configure.php \
+COPY run_openemr.sh auto_configure.php /var/www/localhost/htdocs/openemr/
+COPY utilities/unlock_admin.php utilities/unlock_admin.sh /root/
```

Comment:
not sure that it needs to be in a utilities folder all by its lonesome, since all of the other files there are either config files or utilities. An argument could be made for moving all of them into folders with those names though.

Reply:
I looked at the "fragment of useful PHP" and I had a hunch we'd pick up more of them -- and I wanted them out of the top level so that it'd be a more pleasant read, with fewer moving parts disclosed to a newcomer on first glance.

Reply:
Should we not then also move auto_configure.php in there as well?

Reply:
Not in my view, because it's going to be used most of the time -- I looked at it and thought 'load-bearing', it should be presented to the reader as a thing of value at the top level. If primary install involved a lot of php fragments I'd've considered it differently.

Reply:
As good a reason as any. Everything else looks good to me.

Reply:
@TheToolbox ,
I thought above needing to move auto_configure also, but it was using relative paths, so figured that would then require more testing :)
Also, felt safe since it is always removed.

Reply:
Take that back. It doesn't use relative paths. It could be moved, but again, it is always deleted.